.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Pull request requirements
+    - briefly describe proposed change (What, How, and Why)
```

Comment:
This will be solved by having a pull request template, I think. No need to specify it here.

Reply:
OK, sounds reasonable.